scripts/appleseed.package.py

```diff
@@ -328,15 +329,18 @@ def add_headers_to_stage(self):
 
         # appleseed headers.
         safe_make_directory("appleseed/include")
-        ignore_files = shutil.ignore_patterns("*.cpp", "*.c", "*.xsd", "stdosl.h", "oslutil.h", "as_osl_extensions.h", "snprintf", "version.h.in")
+        ignore_files = shutil.ignore_patterns("*.cpp", "*.c", "*.xsd", "snprintf", "version.h.in")
         shutil.copytree(os.path.join(self.settings.appleseed_headers_path, "foundation"), "appleseed/include/foundation", ignore = ignore_files)
         shutil.copytree(os.path.join(self.settings.appleseed_headers_path, "main"), "appleseed/include/main", ignore = ignore_files)
         shutil.copytree(os.path.join(self.settings.appleseed_headers_path, "renderer"), "appleseed/include/renderer", ignore = ignore_files)
 
-        # OSL headers.
-        shutil.copy(os.path.join(self.settings.appleseed_headers_path, "renderer/kernel/shading/oslutil.h"), "appleseed/shaders/")
-        shutil.copy(os.path.join(self.settings.appleseed_headers_path, "renderer/kernel/shading/stdosl.h"), "appleseed/shaders/")
-        shutil.copy(os.path.join(self.settings.appleseed_headers_path, "renderer/kernel/shading/as_osl_extensions.h"), "appleseed/shaders/")
+    def add_shaders_to_stage(self):
+        progress("Adding shaders to staging directory")
+
+        if os.path.exists("appleseed/shaders"):
```

Comment:
You don't need this test in principle. The appleseed directory is deleted and recreated before packaging start.